doc/md/fundamentals/2-types/1-primitive-types.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 1
+---
+
+# Primitive types
+
+Motoko provides several primitive types that form the foundation of all computations. These include numeric types, characters and text, booleans, and floating-point numbers.
+
+## Numeric types
+
+Motoko supports both signed integers and unsigned naturals. Signed numbers can represent all numbers, positive and negative, while unsigned integers can only represent 0 and positive numbers. Natural numbers are unsigned integers.
+
+- Signed integers: [`Int`](https://internetcomputer.org/docs/motoko/base/Int), [`Int8`](https://internetcomputer.org/docs/motoko/base/Int8), [`Int16`](https://internetcomputer.org/docs/motoko/base/Int16), [`Int32`](https://internetcomputer.org/docs/motoko/base/Int32), [`Int64`](https://internetcomputer.org/docs/motoko/base/Int64)
+- Unsigned naturals: [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat), [`Nat8`](https://internetcomputer.org/docs/motoko/base/Nat8), [`Nat16`](https://internetcomputer.org/docs/motoko/base/Nat16), [`Nat32`](https://internetcomputer.org/docs/motoko/base/Nat32), [`Nat64`](https://internetcomputer.org/docs/motoko/base/Nat64)
+
+The [`Int`](https://internetcomputer.org/docs/motoko/base/Int) and [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) types prevent overflow and underflow since they dynamically expand or shrink as needed.
+
+While [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) can be implicitly converted to [`Int`](https://internetcomputer.org/docs/motoko/base/Int), the reverse is not always true. [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) is a subtype of [`Int`](https://internetcomputer.org/docs/motoko/base/Int). In other words, a [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) value can be used anywhere an [`Int`](https://internetcomputer.org/docs/motoko/base/Int) is expected because [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) is always a valid [`Int`](https://internetcomputer.org/docs/motoko/base/Int). This is why implicit conversions from [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) to [`Int`](https://internetcomputer.org/docs/motoko/base/Int) are allowed.
```

Comment:
```suggestion
Since the set of non-negative integers is a subset the integers, in Motoko,  [`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) is a subtype of [`Int`](https://internetcomputer.org/docs/motoko/base/Int).
This means that every expression of type 
[`Nat`](https://internetcomputer.org/docs/motoko/base/Nat) can implicitly serve as an  `Int` [`Int`](https://internetcomputer.org/docs/motoko/base/Int) without any need for conversion. In Motoko, it is legal to pass a `Nat` to  a function expecting an `Int`, or to assign a `Nat` to a variable storing an `Int`.
```